R/pkg/R/generics.R

```diff
@@ -152,9 +146,9 @@ setGeneric("getNumPartitions", function(x) { standardGeneric("getNumPartitions")
 # @export
 setGeneric("numPartitions", function(x) { standardGeneric("numPartitions") })
 
-# @rdname persist
-# @export
-setGeneric("persist", function(x, newLevel) { standardGeneric("persist") })
+setGeneric("partitionByRDD", function(x, ...) { standardGeneric("partitionByRDD") })
```

Comment:
Is `partitionBy` a problem that the CRAN checks report ? From what I can see we only have a `windowPartitionBy` function which shouldn't collide.